src/Otiport.API/Services/Implementations/MedicineService.cs

```diff
@@ -55,7 +55,7 @@ public async Task<AddMedicineResponse> AddMedicineAsync(AddMedicineRequest reque
         public async Task<DeleteMedicineResponse> DeleteMedicineAsync(DeleteMedicineRequest request)
         {
             var response = new DeleteMedicineResponse();
-            MedicineEntity entity = await _medicineRepository.GetMedicineById(new MedicineEntity() { Id = request.MedicineId });
+            MedicineEntity entity = await _medicineRepository.GetMedicineById(request.MedicineId);
             if (entity != null)
```

Comment:
Burası null değilse değil de null ise deyip StatusCode u Not Found a setleyip return ettir